### DISCUSSION TITLE
In the insert mode, the normal mode will be automatically entered after the text is selected

### DISCUSSION THREAD

**Discussion by bufferstreamer** [JetBrains/ideavim#574] [Q&A]
hi guys, i hava a problem is that when I am in the insert mode, select the text with the mouse or via ctrl+w of idea, the vim will enter the normal mode automatically. This makes it impossible to use some functions of idea such as to add brackets or quotation marks to both ends of the text . i want to know if there are some ways to  control whether to enter the normal mode from the insert mode, rather than the plugin automatically

**Comment by AlexPl292** [comment]
Hi, IdeaVim enters a visual mode, not normal. To keep the regular selection, you can add `set selectmode=ideaselection` to your `~/.ideavimrc`.
